vision/dcgan_mnist/dcgan_mnist.jl

```diff
@@ -105,7 +106,7 @@ function train(; kws...)
     dscr = Discriminator() |> gpu
```

Comment:
define a `device` instead of just using `gpu`?

Reply:
Makes sense